src/index.css

```diff
@@ -1,6 +1,17 @@
-.body{
-    height: 100vh;
-    width: 100vw;
-    background-repeat: no-repeat;
-    background-color: black;
-}
\ No newline at end of file
+/* Can import css files to be used globally. Will use this for text formatting, flex commands and common sizing/indentation */
+@import "./common/styles/fonts.css";
+@import "./common/styles/common.css";
+@import "./common/styles/flex.css";
+
+
+
+/* Create custom colors here. 
+Example use: 
+.className{
+    border: 1px solid var(--blueBorder)
+}
+*/
+
+:root {
+  --blueBorder: #53a5c5;
```

Comment:
We can also create common colors for the entire project. 
They will need to be added here in :root (see existing example)

They are used similar to: 
``` .class {
border: 1px solid var(--blueBorder);
}```
We will want to make sure we keep names consistent. 
Basically anytime you would add a color to a CSS class we should add it here to keep a common theme across the project.